src/platform/options/default_configuration.cpp

```diff
@@ -213,6 +214,8 @@ mo::DefaultConfiguration::DefaultConfiguration(
             "in unexpected ways] throw an exception (instead of a core dump)")
         (debug_opt, "Enable extra development debugging. "
             "This is only interesting for people doing Mir server or client development.")
+        (enable_mirclient_opt, po::value<bool>()->default_value(false),
```

Comment:
Elsewhere you test to see if `enable-mirclient` is supplied (not its value): by providing a default value this guarantees it will always be supplied.